package.json

```
@@ -14,7 +14,8 @@
       "types": "./dist/types/src/Wonka.d.ts",
       "source": "./src/Wonka.ts"
     },
-    "./package.json": "./package.json"
+    "./package.json": "./package.json",
+    "./lib/": "./lib/"
```

This isn't quite complete; we'd also need to add `./src/` since some `.bs.js` files are compiled in-source for this package. However, we can just make this way less restrictive and call it a day:

```suggestion
    "./": "./"
```

---

Yes you're right. I thought compiled js location is controlled local bsconfig, instead it's controlled by user's bsconfig